### ISSUE TITLE
Questions from audience - k2 and Kaldi compatibility 

### ISSUE THREAD

**Issue by leibny** [jtrmal/kaldi2020#32]
Will the current Kaldi toolkit cease its existence and supports?

Is the new platform still compatible to the current one?

**Comment by kkm000** [comment]
> Will the current Kaldi toolkit cease its existence and supports?

No. Until the day there is an easy migration path, Kaldi will be fully supported. New development will slow down, though.

> Is the new platform still compatible to the current one?

This is an unanswerable question, more context is necessary. Define "compatible."

Can you run the same training script? Most probably not. 

Can you replace current production code with the new Kaldi? Certainly yes, although it may not be as simple as “drop in the new code” as it used to. This is one of the goals we keep in mind. There is too many Kaldi uses in production; it would not be right. Kaldi has always been a production-ready codebase, and our goal is to keep it this way. I personally take the responsibility to make the migration path as easy as it could be. To summarize, some assembly will be required, but the tools will come in the package.

If you clarify the question with your use case, and the level of compatibility that you are thinking of, that would be certainly helpful—not only to answer the question, but to define the migration path as well.
